rules/use-mass-transit/rule.md

```diff
@@ -0,0 +1,58 @@
+---
+type: rule
+title: Do you use MassTransit to Build Reliable Distributed Applications?
```

Comment:
hey @danielmackay 

1. Can you remove all the Capitalization as per https://www.ssw.com.au/rules/how-to-capitalize-titles